calculator.go

```diff
@@ -28,17 +29,20 @@ func SizeInBytes(av *types.AttributeValue) (int, error) {
 		}
 		return size, nil
 	case *types.AttributeValueMemberL:
+		// Why is the size 3 here? THis might be worth a comment.
```

Comment:
Good point, it's not very clear from only looking at the code. I'm writing a blog post on this which will explain, but will also add a comment or similar :)

Reply:
Partly addressed here https://github.com/ryeguard/ddbcalc/pull/6